dataprofiler/tests/space_time_analysis/dataset_generation.py

```diff
@@ -51,11 +54,14 @@ def convert_data_to_df(
     :type path: str, None, optional
     :param index: whether to include index in output to csv
     :type path: bool, optional
+    :param column_names: The names of the columns of a dataset
+    :type path: List, None, optional
     :return: a pandas dataframe
     """
     # convert array into dataframe
-    dataframe = pd.DataFrame(np_data)
-
+    if not column_names:
```

Comment:
I think we might be able to simply this by using a different load function.

https://www.geeksforgeeks.org/creating-a-pandas-dataframe-using-list-of-tuples/amp/

Reply:
All of these fixes still require you to submit a column param. I just tried from_items (this one says its not function), from_records (produces same problem as original code unless column param is present), and pivoting seems just as complicated as what im doing now.